security/providers/oidc-common/src/main/java/io/helidon/security/providers/oidc/common/OidcConfig.java

```diff
@@ -2054,5 +2055,13 @@ public Builder clientTimeout(Duration duration) {
         private void clientTimeoutMillis(long millis) {
             this.clientTimeout(Duration.ofMillis(millis));
         }
+
+        private static char[] toCookieEncryptionPassword(Config value) {
+            String cookieEncryptionPasswordValue = value.asString().get();
```

Comment:
the value can never be null - method `get()` would throw an exception if the value is missing

Reply:
empty password is a valid value

Reply:
Ok, will not block empty password then. Or will just remove the condition for empty value.